app/templates/ui/app/create/create.html

```diff
@@ -62,9 +96,17 @@ <h2 class="col-sm-10">Create a Document</h2>
       </div>
     </div>
     <div class="form-group">
-      <label class="col-sm-2 control-label">Email</label>
+      <label class="col-sm-2 control-label required">Email</label>
       <div class="col-sm-10">
-        <input type="email" class="form-control" ng-model="ctrl.person.email" placeholder="">
+        <input type="email" class="form-control" ng-model="ctrl.person.email" placeholder="" name="email" required>
+      </div>
+    </div>
+    <div class="row form-error" ng-if="(createForm.$submitted || createForm.email.$touched) && createForm.email.$error">
+      <div class="col-md-10 col-md-offset-2 has-error has-feedback" >
+        <div ng-messages="createForm.email.$error">
+          <p class="alert alert-danger" ng-message="required">Email is required.</p>
+          <p class="alert alert-danger" ng-message="email">Invalid email format.</p>
+        </div>
       </div>
     </div>
```

Comment:
Come to think of it, it looks like there are a few other input fields with specific types. They are not validated, e.g. you can enter xxx for age..

Maybe also make 'Gender' required? That way things look a bit more realistic..